test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,30 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsNull(Object... objects) {
```

Comment:
Could rethink on the name used for the method. It gives a sense of **only** null instead of **any** null